aspnetcore/mvc/advanced/app-parts.md

```diff
@@ -91,16 +90,39 @@ The `GenericController` class:
 
 [!code-csharp[](./app-parts/sample/AppPartsSample/GenericController.cs?highlight=5-6)]
 
-The result, when a matching route is requested:
+The result when requesting `http://localhost:28431/Sprocket`:
```

Comment:
As we're removing the other screenshot, it makes sense to remove this one. It showed the URL, which is important to the example, so I included that in the text.